radio/src/gui/128x64/model_setup.cpp

```diff
@@ -870,11 +897,41 @@ void menuModelSetup(event_t event)
               s_editMode=0;
             }
 #endif
+#if defined(BINDING_OPTIONS)
+            if (attr && l_posHorz>0) {
+              if(s_editMode>0) {
+                if (l_posHorz == 1) {
+                  if (IS_MODULE_XJT(moduleIdx) && g_model.moduleData[moduleIdx].rfProtocol== RF_PROTO_X16 && s_current_protocol[INTERNAL_MODULE] == PROTO_PXX) {
+                    if(!popupMenuNoItems && !bindingChoiceMade) {
```

Comment:
Why not call this code on `EVT_KEY_PRESS(KEY_ENTER)`? And you set the BIND flag in the menu handler